sdk/core/System.ClientModel/gen/StringBuilderExtensions.cs

```diff
@@ -7,23 +7,23 @@ namespace System.ClientModel.SourceGeneration;
 
 internal static class StringBuilderExtensions
 {
-    public static void AppendLine(this StringBuilder builder, int indent, string value)
+    public static void Ident(this StringBuilder builder, int indent)
     {
         if (indent > 0)
         {
             builder.Append(' ', indent * 4);
         }
+    }
 
+    public static void AppendLine(this StringBuilder builder, int indent, string value)
```

Comment:
nit:
you could revert your change to `Append`, eliminate the `Ident` method and make this:
```c#
builder.Append(builder, indent, value);
builder.AppendLine();
```